InternTrack.Core.Api.Tests.Acceptance/Brokers/InternTrackApiBroker.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using RESTFulSense.Clients;
+
+
```

Comment:
Remove empty space and add copyright header

Reply:
@Manasa1191 take care of this.